src/schema/artwork/index.ts

```diff
@@ -511,6 +511,17 @@ export const artworkFields = () => {
     pickup_available: { type: GraphQLBoolean },
     price: { type: GraphQLString },
     price_currency: { type: GraphQLString },
+    shippingContinentalUsOnly: {
```

Comment:
#minor but I wonder if `shipsToContinentalUSOnly` or even just `shipsToUSOnly` is clearer?

Reply:
I did a lot of waffling on what to name this field and I agree that `shipsToContinentalUSOnly` is clearer than what I currently have, I'll update.